### PR TITLE
Add forum categories

### PR DESCRIPTION
**Pull request by tsteer**

### CODE REVIEW

File: db/migrate/20170831150642_create_forum_categories.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreateForumCategories < ActiveRecord::Migration[5.1]
+  def change
+    create_table :forum_categories do |t|
+      t.string :title
```
Comment (danielburnley) [inline db/migrate/20170831150642_create_forum_categories.rb]: Title doesn't really feel like the right thing to call it - but i can't think of another :man_shrugging:

Comment (craigjbass) [reply db/migrate/20170831150642_create_forum_categories.rb]: display_name ?

File: spec/models/forum_thread_spec.rb [hunk 2]
```diff
@@ -2,16 +2,23 @@
...
       it('is not valid') { is_expected.to_not be_valid }
     end
 
-    context 'with a title' do
+    context 'with a title and no forum category' do
       let(:title) { 'Thread example title' }
+      it('is not valid') { is_expected.to_not be_valid }
+    end
+
+    context 'with a title and a forum category' do
```
Comment (craigjbass) [inline spec/models/forum_thread_spec.rb]: You could nest this in the other context which sets a title which would remove the need to repeat the title let

Comment (tsteer) [reply spec/models/forum_thread_spec.rb]: I tried that earlier and decided to go with this naming method, thanks though!

**Pull request merged**